packages/paste-libraries/data-visualization/src/usePasteHighchartsTheme.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {ThemeContext} from '@twilio-paste/styling-library';
+import type {ThemeShape} from '@twilio-paste/theme';
+import type Highcharts from 'highcharts';
+
+const merge = require('deepmerge');
+
+/**
+ * A hook to apply the Paste theme to Highcharts options.
+ *
+ * @param {Highcharts.Options} options - options to be provided into Highcharts for rendering.
+ * @returns {Highcharts.Options} optionsWithPasteTheme - Passed options with deepmerged Paste Theme
+ */
+export const usePasteHighchartsTheme = (options: Highcharts.Options): Highcharts.Options => {
```

Comment:
We need this to be a React hook so that the styles update upon toggling between light and dark theme. Can't just export a static object, unless we instruct developers to swap it themselves which is much less ergonomic and more error prone.